### PR TITLE
Edit file index.rst

### PR DESCRIPTION
**Pull request by fatData**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -49,32 +49,32 @@ features:
 * Integrated access to the pdb debugger and the Python profiler.
 
 
-The Command line interface inherits all the above functionality and posses 
+The Command line interface inherits all of the above functionalities and possess
```
Comment (takluyver) [inline docs/source/index.rst]: I think 'functionality' is better here - it functions as an uncountable noun.

File: docs/source/index.rst [hunk 2]
```diff
@@ -49,32 +49,32 @@ features:
...
 to improve your Python workflow.
 
-You can always find the table of content for this documentation in the left
-sidebar, allowing you to come back on previous section if needed, or skip ahead. 
+You can find the table of content for this documentation in the left
+sidebar, allowing you to come back to previous sections or skip ahead, if needed. 
 
 
-The latest development version is always available from IPython's `GitHub
+The latest development version is always available through IPython's `GitHub
```
Comment (takluyver) [inline docs/source/index.rst]: I prefer 'from' a repository.

File: docs/source/index.rst [hunk 3]
```diff
@@ -98,7 +98,7 @@ repository <http://github.com/ipython/ipython>`_.
 .. seealso::
 
    `Jupyter documentation <http://jupyter.readthedocs.io/en/latest/>`__
-     The Notebook code and many other pieces formerly in IPython are now parts
+     The Notebook code and other pieces formerly in IPython, are now part
```
Comment (takluyver) [inline docs/source/index.rst]: I think this either needs no commas or two (the other one being after 'code'), not one.

**Comment by takluyver** [comment]
Thanks; i've pointed out a couple of places where I think the previous wording was preferable, but this is an improvement overall.

File: docs/source/index.rst (hunk 1 above)
Comment (willingc) [inline docs/source/index.rst]: I would agree with `functionality` and swap out `possess` for `adds`. `...inherits the above functionality and adds...`

File: docs/source/index.rst (hunk 2 above)
Comment (willingc) [inline docs/source/index.rst]: I agree. Let's leave `from` here.

File: docs/source/index.rst (hunk 3 above)
Comment (willingc) [inline docs/source/index.rst]: `The Jupyter documentation provides information about the Notebook and other Project Jupyter projects`.

**Comment by fatData** [comment]
I agree. I will revert those changes.

**Review by willingc — APPROVED** [review]
Looks good to me. Thanks for taking the time to submit the PR @fatData.

**Review by wrt2 — COMMENTED** [review]
#6FCFF7

**Pull request merged**